maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -140,9 +146,14 @@ public class MavenCli {
 
     private static final String EXT_CLASS_PATH = "maven.ext.class.path";
 
-    private static final String EXTENSIONS_FILENAME = ".mvn/extensions.xml";
+    private static final String DOT_MVN = ".mvn";
 
-    private static final String MVN_MAVEN_CONFIG = ".mvn/maven.config";
+    private static final String UNABLE_TO_FIND_ROOT_PROJECT_MESSAGE = "Unable to find the root directory. Create a "
+            + DOT_MVN + " directory in the project root directory to identify it.";
+
+    private static final String EXTENSIONS_FILENAME = DOT_MVN + "/extensions.xml";
```

Comment:
That's not a filename anymore, but a path.

Reply:
Before and after this PR the contents of `EXTENSIONS_FILENAME` is unchanged, so I take this as nitpicking :smile:

Reply:
Shit, you are right. It just popped my eye.